### DISCUSSION TITLE
php7.1-redis is not found

### DISCUSSION THREAD

**Discussion by karmendra** [oerdnj/deb.sury.org#2087] [General]
I added the ppa:ondrej/php repository on Ubuntu 18.04 (Bionic) and attempted to install php7.1-redis, but encountered the error: `E: Unable to locate package php7.1-redis`.

When attempting to install `php7.2-redis`, it suggested installing php-redis instead, which worked fine. However, the same recommendation wasn't provided for php7.1-redis.

This issue is causing several of my scripts to fail, and unfortunately, I'm unable to access these scripts to make necessary modifications.

Is there a workaround to locally map `php7.1-redis` to `php-redis`, so that attempting to install php7.1-redis triggers the same message as `Note, selecting 'php-redis' instead of 'php7.1-redis'`?

**Comment by karmendra** [comment]


**Reply by oerdnj** [reply]
> Thanks to ChatGPT

Congratulations, you and ChatGPT posted absolute crap! Well done!

**Reply by karmendra** [reply]
I respect your work a lot, so won't attempt to dis you with any response to your sarcasm.
I respect peoples opinion about ChatGPT, its some what similar to how it was with Google search 15 year back.

P.S.  Later found that your ppa doesn't support unsupported Ubuntu Version 18.04, so I decide to upgrade the OS to Ubuntu 20.04 anyways.

**Reply by oerdnj** [reply]
I deleted the ChatGPT generated crap, so other people are not misled by the nonsenses you posted here.

**Reply by karmendra** [reply]
Thanks, have a good day.
